website/content/docs/drivers/exec.mdx

```diff
@@ -188,6 +188,30 @@ able to make use of IPC features, like sending unexpected POSIX signals.
 undesirable consequences, including untrusted tasks being able to compromise the
 host system.
 
+- `denied_host_uids` - (Optional) Specifies a comma-separated list of host uids to
+  deny. Ranges can be specified by using a hyphen separating the two inclusive ends.
+  If a "user" value is specified in task configuration and that user has a user id in
+  the given ranges, the task will error before starting. This will not be checked on Windows
+  clients.
+
+```hcl
+config {
+  denied_host_uids = "0,10-15,22"
+}
+```
+
+- `denied_host_gids` - (Optional) Specifies a comma-separated list of host uids to
+  deny. Ranges can be specified by using a hyphen separating the two inclusive ends.
+  If a "user" value is specified in task configuration and that user has is part of
+  any groups with gid's in the specified ranges, the task will error before
+  starting. This will not be checked on Windows clients.
```

Comment:
```suggestion
- `denied_host_gids` - (Optional) Specifies a comma-separated list of host gids to
  deny. Ranges can be specified by using a hyphen separating the two inclusive ends.
  If a "user" value is specified in task configuration and that user is part of
  any groups with gid's in the specified ranges, the task will error before
  starting. This will not be checked on Windows clients.
```